Source/Fuse.Elements/Element.Drawing.uno

```diff
@@ -66,21 +66,17 @@ namespace Fuse.Elements
 			if (Visibility != Visibility.Visible)
 				return;
 
+			var visibleRect = GetVisibleViewportInvertPixelRect(dc, RenderBoundsWithEffects);
+			if (visibleRect.Size.X == 0 || visibleRect.Size.Y == 0)
+				return;
+
 			extern double t;
 			if defined(FUSELIBS_PROFILING)
 			{
 				t = Uno.Diagnostics.Clock.GetSeconds();
 				Fuse.Profiling.BeginRegion(this.ToString());
 			}
 
-			var visibleRect = GetVisibleViewportInvertPixelRect(dc, RenderBoundsWithEffects);
-			if (visibleRect.Size.X == 0 || visibleRect.Size.Y == 0)
-			{
-				if defined(FUSELIBS_PROFILING)
-					Fuse.Profiling.EndRegion(Uno.Diagnostics.Clock.GetSeconds() - t);
-				return;
-			}
-
```

Comment:
I think you have no idea how expensive reading `RenderBoundsWithEffects` can be (worst case is full subtree traversal, ). This is very useful information, and you don't want to get that reported on the wrong element. That makes your data tell you the wrong story.

I already suggested an alternative approach that avoids the bulk of the profiling-cost and allows you to catch the cases where this ends up taking a significant amount of time.

Another alternative is to add profiling inside `CalcRenderBoundsWithEffects()`, which will only trigger when the cache invalid.